### ISSUE TITLE
make logo 128x128

### ISSUE THREAD

**Issue by fee1-dead** [yarl/pattypan#167]
linux needs logos to be powers of 2 and the current dimensions (100x100) do not work.

**Comment by Abbe98** [comment]
Do you have any more details about this? In Gnome and in Ubuntu Gnome environments the logo is known to work.

**Comment by fee1-dead** [comment]
Thanks for the quick response. I think I might have done something wrong when I checked, but I believe the icons are preferred to have a size that is a power of 2. `man xdg-icon-resource` does say that common dimensions include 128x128.

**Comment by Abbe98** [comment]
I'm manly familiar with Gnome they try to draw logos as 128px from whatever they get. The preference is SVG([Gnome HIG](https://developer.gnome.org/hig/guidelines/app-icons.html)).

Is there a DE which fails to render the Pattypan logo?

**Comment by fee1-dead** [comment]
I am on KDE and I have `share/icons/hicolor/100x100/apps/pattypan.png`. this directory works for icons with other sizes.

**Comment by Abbe98** [comment]
We have the logo in SVG here: https://commons.wikimedia.org/wiki/File:Pattypan.svg

And considering that 128 is a more common size I agree that we should replace it, but we should probably test such a change across all platforms.

**Comment by fee1-dead** [comment]
If possible, I think there should be an svg of the logo in this repository? that would make packaging easier.

**Comment by Abbe98** [comment]
Indeed, especially useful if someone would like to do a Flatpak or similar.
